src/scripts/components/widgets/modals/ModalAskPassword.js

```diff
@@ -101,7 +104,6 @@ class ModalAskPassword extends Component<Props, Object> {
         },
         'error'
       )
-      throw err
```

Comment:
@eliawk I don't _think_ we want to throw errors here; this is polluting the browser console and we are, in fact, handling these errors. What are your thoughts?

Reply:
yes sounds good to me